src/routes/register/registerDTO.ts

```diff
@@ -0,0 +1,37 @@
+import {IsEnum, IsString, Length, Matches} from 'class-validator'
+
+enum TGender {
+	Male = 'Male',
+    Female = 'Female',
+    Other = 'Other'
+}
+
+interface Form {

```

Comment:
Czemu to nadal się Form nazywa? Ta nazwa nie zdradza co może się kryć pod zmienną